src/pages/admin/student/edit/pageContainer.tsx

```diff
@@ -0,0 +1,17 @@
+import { connect } from 'react-redux';
```

Comment:
Pending to add spec to `pageContainer.tsx` remember [TDD work flow](https://github.com/Lemoncode/LeanMood/wiki/devworkflow#unit-testing--tdd)